apps/mobile/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,73 @@
+import LibraryCheckbox, {
+	type CheckBoxProps,
+} from "@react-native-community/checkbox";
+import React from "react";
+
+import { LinearGradient, Text, View } from "~/libs/components/components";
+import { AngleGradient, BaseColor, GradientColor } from "~/libs/enums/enums";
+import { globalStyles } from "~/libs/styles/styles";
+
+import { styles } from "./styles";
+
+type Properties = {
+	hasVisuallyHiddenLabel?: boolean;
+	isChecked: boolean;
+	label: string;
+	onValueChange: (newValue: boolean) => void;
+} & CheckBoxProps;
```

Comment:
In case that you don't use other props from original `CheckBoxProps` remove it
```suggestion
type Properties = {
	hasVisuallyHiddenLabel?: boolean;
	isChecked: boolean;
	label: string;
	onValueChange: (newValue: boolean) => void;
};
```